packages/tasks/src/tasks/text-generation/spec/input.json

```
@@ -88,6 +93,33 @@
 					"description": "Watermarking with [A Watermark for Large Language Models](https://hf.co/papers/2301.10226)"
 				}
 			}
+		},
+		"GrammarType": {
+			"title": "GrammarType",
```

(needed for type names to be prefixed by `TextGeneration`)
```suggestion
			"title": "TextGenerationGrammarType",
```